codes/zig/chapter_dynamic_programming/climbing_stairs_dfs_mem.zig

```diff
@@ -22,7 +22,7 @@ fn dfs(i: usize, mem: []i32) i32 {
 }
 
 // 爬楼梯：记忆化搜索
-fn climbing_stairs_dfs_mem(comptime n: usize) i32 {
+fn climbingStairsDfsMem(comptime n: usize) i32 {
```

Comment:
climbingStairsDFSMem

Same as others. Please refer to Java code's naming style